plugins/notifications-backend/src/service/router.ts

```diff
@@ -19,6 +19,10 @@ import {
 } from './permissions';
 import { RouterOptions } from './types';
 
+// TODO: fix filename resolving
```

Comment:
This is a hot-fix. We will search for a better way

Reply:
the file name was hard coded and so it should be. the backend is tied tightly to the spec.

Reply:
We need to improve, so far it's still a hotfix, we need to have it working by tomorrow.
The former path worked well when used in the development wrapping backstage instance. But was not matching when the plugin is added as an NPM dependency to node_modules (which is are ultimate goal).

Reply:
that does not explain the technical problem. why should npm package care where my file is stored? i provide a folder with files and it should package it, no?

Reply:
Note: the Janus NPM packages have been published, so for sure we don't need my provate account mentioned bellow